src/Components/TouristInfoCards.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const TouristInfoCards = ({ image, title = 'hotel dami', text, link }) => {
```

Comment:
Given it's a generic element, the title shouldn't have a specific default like this